web_decimal_numpad_dot/static/src/js/numpad_dot.js

```diff
@@ -0,0 +1,75 @@
+/* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_decimal_numpad_dot.FieldFloat', function (require) {
+"use strict";
+
+    var core = require('web.core');
+    var form_widgets = require('web.form_widgets');
+    var translation = require('web.translation');
+
+    var FieldFloatDecimalDot = form_widgets.FieldFloat.extend({
```

Comment:
What you want to do here is to extend the behavior of core `FieldFloat` widget, right? Then you should use `.include` (add behavior to current prototype) instead of `.extend` (create a new prototype with extra behavior).